test/functional/apps/management/_scripted_fields.js

```diff
@@ -56,13 +56,13 @@ bdd.describe('creating and using Lucence expression scripted fields', function d
     await PageObjects.common.navigateToApp('discover');
     await PageObjects.common.debug('setAbsoluteRange (' + fromTime + ') to (' + toTime + ')');
     await PageObjects.header.setAbsoluteRange(fromTime, toTime);
-    await PageObjects.header.isGlobalLoadingIndicatorHidden();
+    await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
It already does wait in most places, `setAbsoluteRange` amongst them. I'll try to clean that up tomorrow morning.